Support for --strict

When building a project with generated api and --strict, the build fails with: 

api/base-service.ts(38,11): error TS2564: Property '_rootUrl' has no initializer and is not definitely assigned in the constructor.

--- a/templates/baseService.mustache
+++ b/templates/baseService.mustache
@@ -35,7 +35,7 @@ export class BaseService {
   ) {
   }
 
-  private _rootUrl: string;
+  private _rootUrl: string = '';
 
   /**
    * Returns the root url for API operations. If not set directly in this

